Heroku deployment failing

The Heroku pipeline has been failing since Lerna 2.0.0-rc.1 was brought in.

```
       Lerna v2.0.0-rc.1
       Independent Versioning Mode
       This is not a git repository, did you already run `git init` or `lerna init`?
       
       npm ERR! Linux 3.13.0-112-generic
       npm ERR! argv "/tmp/build_6b2b2babbbb3cc8f0a897e4e530ce41d/nteract-commuter-61cfbaf/.heroku/node/bin/node" "/tmp/build_6b2b2babbbb3cc8f0a897e4e530ce41d/nteract-commuter-61cfbaf/.heroku/node/bin/npm" "run" "lerna"
       npm ERR! node v6.9.5
       npm ERR! npm  v3.10.10
       npm ERR! code ELIFECYCLE
       npm ERR! @nteract/commuter-client@0.1.0 lerna: `lerna bootstrap`
       npm ERR! Exit status 1
       npm ERR!
       npm ERR! Failed at the @nteract/commuter-client@0.1.0 lerna script 'lerna bootstrap'.
       npm ERR! Make sure you have the latest version of node.js and npm installed.
       npm ERR! If you do, this is most likely a problem with the @nteract/commuter-client package,
       npm ERR! not with npm itself.
       npm ERR! Tell the author that this fails on your system:
```

It looks like lerna is not pleased with the fact that it isn't a git repo by the time it reaches Heroku.

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "clean:build": "lerna exec -- rm -rf ./build",
     "format:code": "prettier --write 'packages/*/src/**/*.js' 'packages/*/test/**/*.js'",
     "install": "npm run lerna && npm run build",
-    "lerna": "lerna bootstrap",
+    "lerna": "lerna init && lerna bootstrap",
     "publish": "npm run clean && npm install && lerna publish",
     "start": "lerna exec --scope @nteract/commuter-cli -- npm run start",
     "start:watch": "concurrently \"lerna exec --scope @nteract/commuter-cli -- npm run start\" \"npm run watch\"",
@@ -43,7 +43,7 @@
   },
   "homepage": "https://github.com/nteract/commuter#readme",
   "dependencies": {
-    "lerna": "2.0.0-rc.1"
+    "lerna": "^2.0.0-rc.1"
   },
   "devDependencies": {
     "concurrently": "^3.1.0",

